ceph-dev/config/definitions/ceph-dev.yml

```diff
@@ -59,6 +59,26 @@ If this is checked, then the binaries will be built and pushed to chacra even if
           default: "default"
           description: "Type of Ceph build, choices are: notcmalloc, default (i.e. with tcmalloc). Defaults to: 'default'"
 
+      - string:
```

Comment:
why isn't this a boolean instead?

Reply:
`description: 'Build container with development release of Ceph.  Note: this must be "false" or "true" so that it can execute a command or satisfy a string comparison'`

Reply:
I am aware that the intention here was to make it easier to compare it later as a string in BASH. I am asking here because Jenkins canl set this to a real boolean which is used in other jobs. For example, in ceph-tag we have the following:

```bash
if [ "$TAG" = false ] ; then
    echo "Assuming tagging process has succeeded before because TAG was set to false"
    exit 0
fi
```

And `TAG` is created as a boolean paramater in the YAML file:

```yaml
      - bool:
          name: TAG
          description: "When this is checked, Jenkins will remove the previous private tag and recreate it again, changing the control files and committing again. When this is unchecked, Jenkins will not do any commit or tag operations. If you've already created the private tag separately, then leave this unchecked.
Defaults to checked."
          default: true
```

Using a string that looks like a boolean but isn't a real boolean so that scripts can compare it to the contents of the string is going to be misleading if others are trying to update or modify these scripts at a later point.

Reply:
I don't disagree with you.  But presumably you already argued this with @dmick and ended up being okay with it: https://github.com/ceph/ceph-build/pull/1378

I just copied/pasted from the ceph-dev-new job.

Reply:
@alfredodeza I changed it to a bool and tested it.

true: https://jenkins.ceph.com/job/ceph-dev-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos7,DIST=centos7,MACHINE_SIZE=huge/34151/

false: https://jenkins.ceph.com/job/ceph-dev-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos7,DIST=centos7,MACHINE_SIZE=huge/34152/